### ISSUE TITLE
Open block can break highlighting.

### ISSUE THREAD

**Issue by lucascosti** [asciidoctor/atom-language-asciidoc#148]
## Description

I am also experiencing an open block breaking highlighting. the `--` in the example ends the highlighting.

## Screenshots

![asciidoc-language-bug](https://cloud.githubusercontent.com/assets/4434330/15642674/3c8d67fc-268c-11e6-8511-5c2a102b81a8.png)

## Syntax example

Your syntax example.

```
[[zip_rollback_management_cli]]
.Rolling Back a Patch Using the Management CLI
. From the management CLI, use the `patch history` command to find the ID of the patch that you want to roll back.
+
--
NOTE: If you are using a managed domain, you must add the `--host=_HOSTNAME_` argument to the commands in this procedure to specify the {ProductShortName} host.
--

. Roll back the patch with the appropriate patch ID from the previous step.
+
[options="nowrap"]
----
patch rollback --patch-id=PATCH_ID --reset-configuration=TRUE
----
+
The patch tool will warn if there are any conflicts in attempting to roll back the patch. If there are conflicts, enter `patch --help` for the available arguments to re-run the command with an argument specifying how to resolve the conflicts.

.  Restart the {ProductShortName} server for the patch roll back to take effect:
+
[options="nowrap"]
----
shutdown --restart=true
----
```

**Comment by ldez** [comment]
Thanks for reporting here. :+1: 

It's fixed in #149

**Comment by lucascosti** [comment]
Thanks @ldez, love your work! :smile:
